package.json

```diff
@@ -31,10 +31,9 @@
   "scripts": {
     "prestart": "export BROWSER='firefox'",
     "start": "react-app-rewired start",
-    "prebuild": "rm -rf build && mkdir build",
+    "prebuild": "mkdir -p build",
```

Comment:
I don't think deleting the previous build is necessary, and helps give you a sense of improvement on build.